securedrop/store.py

```diff
@@ -201,25 +201,36 @@ def get_bulk_archive(self,
                        for i in selected_submissions])
         # The below nested for-loops are there to create a more usable
         # folder structure per #383
+        missing_files = False
+
         with zipfile.ZipFile(zip_file, 'w') as zip:
             for source in sources:
                 fname = ""
                 submissions = [s for s in selected_submissions
                                if s.source.journalist_designation == source]
                 for submission in submissions:
                     filename = self.path(submission.source.filesystem_id, submission.filename)
-                    document_number = submission.filename.split('-')[0]
-                    if zip_directory == submission.source.journalist_filename:
-                        fname = zip_directory
+
+                    if os.path.exists(filename):
```

Comment:
Nitpick: in the previous file you're using pathlib (which is more modern), but not in this file.

Reply:
I have a preference for `pathlib` but this function uses `os.path` functions in multiple places so I stayed with that for consistency... Don't know if that's a valid reason though ;)

Reply:
If it were up to me (it's not)... I would use pathlib for all new code. But your reason for not doing so is valid too haha.